### DISCUSSION TITLE
hiding C# doesn't work?

### DISCUSSION THREAD

**Discussion by abhikatta** [anuraghazra/github-readme-stats#2645] [Q&A] [answered]
Hi, how do i hide C# language in the language card? I tried setting the parameter to :

1. hide=C#
2. hide=c#
3. hide=cs
4. hide=c-sharp
5. hide=C-Sharp
but none seemed to work. Is it a bug or am i doing it wrong? If wrong, please let me know the right way. Thank you.

**Comment by qwerty541** [comment, marked as answer]
Try `c%23`

Language names should be URI-escaped, as specified in [Percent Encoding](https://en.wikipedia.org/wiki/Percent-encoding) (i.e: c++ should become `c%2B%2B`, jupyter notebook should become `jupyter%20notebook`, etc.) You can use [urlencoder.org](https://www.urlencoder.org/) to help you do this automatically.

**Reply by abhikatta** [reply]
Thank you, it helps.
